plugins/performance-lab/load.php

```diff
@@ -5,7 +5,7 @@
  * Description: Performance plugin from the WordPress Performance Team, which is a collection of standalone performance features.
  * Requires at least: 6.5
  * Requires PHP: 7.2
- * Version: 3.4.1
+ * Version: 3.5.0
```

Comment:
Can we bump it to `3.4.2` since there aren't any major changes in this release?

Reply:
I debated this but according to semver a minor version bump includes any enhancements that aren't back-compat breaks. A patch version is just for bug fixes only. So since this version includes just enhancements as well as a new "feature" for listing Web Worker  Offloading, I think a minor version bump is warranted.

Reply:
Agreed, as soon as something includes enhancements, it should be a minor version bump.